### PR TITLE
TX_POWER is an error

### PR DESCRIPTION
**Pull request by mawdegroot**
TX_POWER is an error when there is no lower or equal power level available.
TX_POWER is a warning when there is a lower or equal power level available.

### CODE REVIEW

File: lora_pkt_fwd/src/lora_pkt_fwd.c [hunk 1]
```diff
@@ -2596,17 +2600,20 @@ void thread_down(void) {
             if (jit_result == JIT_ERROR_OK) {
                 uint8_t tx_lut_idx = 0;
                 i = get_tx_gain_lut_index(txpkt.rf_power, &tx_lut_idx);
-                if ((i < 0) || (txlut.lut[tx_lut_idx].rf_power != txpkt.rf_power)) {
+                if (i < 0) {
+                    jit_result = JIT_ERROR_TX_POWER;
+                    printf("WARNING: No TX power lower or equal to %ddBm is supported, rejecting transmission\n", txpkt.rf_power);
```
Comment (JayKickliter) [inline lora_pkt_fwd/src/lora_pkt_fwd.c:2605]: haven't these values been converted from `ddBm` (really `cBm`) to `dBm` by this point?

Comment (mawdegroot) [reply lora_pkt_fwd/src/lora_pkt_fwd.c:2605]: Correct, it is in `dBm`, the first `d` is part of the `%d` format string.

Comment (JayKickliter) [reply lora_pkt_fwd/src/lora_pkt_fwd.c:2605]: Hah, of course. Sorry about that

Comment (JayKickliter) [reply lora_pkt_fwd/src/lora_pkt_fwd.c:2605]: can you repoint the submodule to the merged helium/lora_gateway sha?

**Comment by JayKickliter** [comment]
can you repoint the submodule to the merged helium/lora_gateway sha?

**Comment by mawdegroot** [comment]
it should point to the latest helium/lora_gateway sha now.

**Pull request merged**